src/api/dependency-discovery/src/github.js

```diff
@@ -0,0 +1,60 @@
+const { request } = require('@octokit/request');
+
+const hasGitHubExpired = (dateTime) => Date.now() - dateTime > 60 * 60 * 1000;
+
+const gitHubInformationRequestor = {};
+
+async function constructGitHubRequest(packageName, gitHubUrl) {
+  const labelsToSearchFor = ['hacktoberfest', 'good first issue', 'help wanted'];
```

Comment:
`labels` is probably good enough, I think it's good to avoid long variable names if possible